webpack/webpack.config.js

```diff
@@ -68,6 +68,19 @@ const yoastExternals = {
 	"@yoast/style-guide": "window.yoast.styleGuide",
 };
 
+const yoastExternalsMapping = {
```

Comment:
> What's the difference between `yoastExternals` and `yoastExternalsMapping`? 
...is what everyone will ask. Which is a clear signal the solution that follows is probably not simple enough.